tuning/sft_trainer.py

```diff
@@ -83,6 +90,8 @@ def train(
                               or TrainerControllers. Callbacks associated with \
                               tracker with automatically be added.
         exp_metadata: Dict of key value pairs passed to train to be recoreded by the tracker.
+        quantized_lora_config: tuning.config.acceleration_configs.QuantizedLoraConfig \
```

Comment:
some questions I have:

since this requires augmentation and peft_config , is quantized_lora_config only expected to work with LoRA tuning and Loraconfig ,  or can one also apply it with fine tuning and prompt tuning?

I dont see any checks saying only if LorAconfig can apply this plugin